ignite/services/plugin/plugin.go

```diff
@@ -236,6 +236,23 @@ func (p *Plugin) load(ctx context.Context) {
 	}
 
 	if p.IsLocalPath() {
+		// if directory is relative, make it absolute
+		if !filepath.IsAbs(p.srcPath) {
+			wd, err := os.Getwd()
+			if err != nil {
+				p.Error = errors.Errorf("failed to get working directory: %w", err)
+				return
+			}
+
+			srcPathAbs, err := filepath.Abs(p.srcPath)
+			if err != nil {
+				p.Error = errors.Errorf("failed to get absolute path of %s: %w", p.srcPath, err)
+				return
+			}
+
+			p.srcPath = filepath.Join(wd, srcPathAbs)
```

Comment:
Only the `srcPathAbs, err := filepath.Abs(p.srcPath)` does not work? It is just a corner case: if the plugin does not live in the same partition, it may not work

Reply:
It won't work as it is a relative path: https://go.dev/play/p/UAEkQO40IRg. We want the absolute path from the root, not form the current directory

Reply:
You were right actually, Abs does a GetWd as well.